.editorconfig

```
@@ -0,0 +1,28 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+
+trim_trailing_whitespace = true
+insert_final_newline = true
```

The root of this repository doesn't use much of the other file extension types, but it looks like this should cover the existing style for `.yml` and `.sh` files. I would suggest that this might be better limited in scope instead:

```suggestion
[{*.sh,*.yml}]
indent_style = space
indent_size = 2
end_of_line = lf
charset = utf-8

trim_trailing_whitespace = true
insert_final_newline = true
```